irods/library/irods_unixfilesystem_resource

```diff
@@ -16,10 +16,10 @@ short_description: Create an iRODS Unix filesystem storage resource
 
 version added: "2.11.11"
 
-description: An ansible module for creating an iRODS Unix filesystem storage
-resource
+description: 
+    - An ansible module for creating an iRODS Unix filesystem storage resource
 
-required variables:
+options:
     name:
         description:
             - the name of the resource
```

Comment:
If the string fits on the same line as the field, put it on that line. This comment applies to all of the `description` fields in this section.